public/utils.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Method to escape and sanitize string and handling meta-characters in jquery
+ * @param {string} name String of room name, or any other name
+ */
+const convertIntoId = (name) => name.replace(/[!"#$%&'()*+,.\/:;<=>?@[\\\]^{|}~ ]/g, "\\$&");
+
+
+/**
+ * Method to convert an array into an unordered list
+ * @param {array} arr Array of items
+ */
+const convertIntoList = (arr) => {
+  let list = ('<ul>');
+  for(let i=0; i<arr.length; i++)	list = list.concat(`<li>${arr[i]}</li>`);
+  list = list.concat('</ul>');
+  return list;
+}
```

Comment:
You should add all these functions in `script.js` and rename that to `utils.js`.

Reply:
I'm not understanding what you're asking. If there are methods that should go in `utils.js`, please let me know. I don't think everything in `script.js` is a utility function, is it?

Reply:
At present, there are few lines of code in `script.js` which are not utility function(that's why I created #117), but that file is there just to add utility functions.

Reply:
Again, we're introducing scope creep. I pulled out some methods into the `utils.js` file because they seemed re-usable. If you want all of them in a separate file, then another issue should be made for this, IMO. It has nothing to do with the welcome bot.